apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -108,6 +115,22 @@ export default class PublishAction extends BaseRushAction {
       `the package is older than the current latest, so in publishing workflows for older releases, providing ` +
       `a tag is important. When hotfix changes are made, this parameter defaults to 'hotfix'.`
     });
+
+    // NPM pack tarball related parameters
+    this._pack = this.defineFlagParameter({
+      parameterLongName: '--pack',
+      description:
+        `Packs projects into tarballs instead of publishing to npm repository. It can only be used when ` +
+        `--include-all is specified. If this flag is specified, NPM registry related parameters will be ignored.`
+    });
+    this._releaseFolder = this.defineStringParameter({
+      parameterLongName: '--release-folder',
+      key: 'RELEASEFOLDER',
+      description:
+      `This parameter is used with --pack parameter to provide customized location for the tarballs instead of ` +
+      `the default value. `
+    });
```

Comment:
Why does this need to be a separate option?  Maybe it could just be `rush publish --pack some/folder`